src/fXmlRpc/Parser/NativeParser.php

```diff
@@ -26,13 +26,12 @@
 use DateTime;
 use DateTimeZone;
 use fXmlRpc\Exception\MissingExtensionException;
+use fXmlRpc\Exception\ResponseException;
```

Comment:
I wonder if this should now be a ParseException, as the parser shouldn't need to know anything about the response